python/format/README.md

```diff
@@ -0,0 +1,22 @@
+# Format
+
+## Install
+
+Using Python>=3.8:
+
+```bash
+python -m pip install .
+```
+
+## Validate a file
+
+```bash
+python scripts/validate.py --file ../../datasets/titanic/metadata.json
+```
+
```

Comment:
Maybe add a sentence on what the behaviour is when the config is valid or invalid: return code is 0 if no errors, otherwise return code is 1 and the first error is displayed, or something alike.

Reply:
Done.